SerpentComparison.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,8 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		245AA8C61E6EF3E000C91BD7 /* Marshal.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 245AA8C51E6EF3E000C91BD7 /* Marshal.framework */; };
+		245AA8C71E6EF3E000C91BD7 /* Marshal.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 245AA8C51E6EF3E000C91BD7 /* Marshal.framework */; };
```

Comment:
Somehow the file got added twice. Can you double check it's not added twice in the build phases? :)